src/main/resources/userEmailTemplate-feedbackSessionClosing.html

```diff
@@ -56,10 +59,15 @@
 </p>
 
 <p>
+    For any non-technical queries (e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.), you can contact your instructor(s) at: ${coOwnersEmails}
+</p>
+
+<p>
+    If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
```

Comment:
@leeyimin apologies for the extremely delayed response! Back to full-time contribution now 😀. Requesting clarification on which line is duplicated here?